__tests__/app.test.js

```diff
@@ -73,7 +73,49 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/banana")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("Bad Request");
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+});
+
+// 4. GET /api/articles
+
+describe("GET /api/articles", () => {
+  test("1. GET: 200 Returns an array of article objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(5);
```

Comment:
Great that you're checking length to avoid false positives 👍 However there should be more articles than this (about 13 in total I believe?) - I think it's likely that you may have used a SQL JOIN which excludes articles that don't have any comments, which isn't what we want so please have a look at that 👀